CharlieBackend.Api/Middlewares/ExceptionHandleMiddleware.cs

```diff
@@ -52,6 +52,9 @@ private Task HandleExceptionAsync(HttpContext context, Exception exception)
                 case nameof(EntityValidationException):
                     context.Response.StatusCode = 400;
                     break;
+                case nameof(LessonNotDoneException):
+                    context.Response.StatusCode = 403;
```

Comment:
return 400 error.
Add this to appropriate case

Reply:
Fixed
![Screenshot_14](https://user-images.githubusercontent.com/56502713/112471813-0769db00-8d75-11eb-86ba-172973f898ed.png)